src/OpenTelemetry.Instrumentation.AdditionalDeps/OpenTelemetry.Instrumentation.AdditionalDeps.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
+  </ItemGroup>
+
+  <Target Name="CallComposeStore" AfterTargets="Publish">
+    <PropertyGroup>
+      <AdditionalDepsPath>$(OutputDir)\AdditionalDeps</AdditionalDepsPath>
+      <StoreOutputBasePath>$(OutputDir)\store\</StoreOutputBasePath>
+      <AdditionalDepsPath Condition="'$(TracerHomePath)' != ''">$(TracerHomePath)\AdditionalDeps</AdditionalDepsPath>
+      <StoreOutputBasePath Condition="'$(TracerHomePath)' != ''">$(TracerHomePath)\store</StoreOutputBasePath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <StoreRuntime Include="x86">
+        <PlatformTargetForStore>x86</PlatformTargetForStore>
```

Comment:
nit: since it is later used with the prefix `StoreRuntime` shorter names could be used, e.g. `Platform` and `Path`